inst/rmarkdown/templates/jds/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,271 @@
+
+A professional table contains no vertical lines.
+
+Table \ref{tab:realdata} is an illustration.
+
+\begin{table}[tbp]
+  \caption{Analysis results for real data. Point
+    estimates (EST) from both two-piece method and marginal method are
+    reported. Standard error of point estimates are evaluated by
+    parametric bootstrap (SE).}%
+  \label{tab:realdata}
+  \centering
+  \begin{tabular}{llrrrr}
+    \toprule
+    \multicolumn{1}{l}{Season} & \multicolumn{1}{l}{Parameter}
+                               & \multicolumn{2}{c}{Two-piece method} & \multicolumn{2}{c}{Marginal method}                         \\
+    \cmidrule(r){3-4} \cmidrule(r){5-6}
+                               &                                      & EST                                 & SE    & EST   & SE    \\
+    \midrule
+    Summer                     & $\lambda_1$                          & 2.841                               & 0.459 & 1.090 & 0.280 \\
+                               & $\lambda_0$                          & 0.179                               & 0.014 & 0.158 & 0.015 \\
+                               & $\sigma$                             & 1.335                               & 0.106 & 0.999 & 0.104 \\
+                               & $\sigma_\epsilon (\times 10^{-2})$   & 1.854                               & 0.087 & 1.879 & 0.078 \\
+    Winter                     & $\lambda_1$                          & 6.225                               & 0.825 & 4.720 & 0.711 \\
+                               & $\lambda_0$                          & 0.118                               & 0.010 & 0.114 & 0.009 \\
+                               & $\sigma$                             & 1.506                               & 0.095 & 1.454 & 0.089 \\
+                               & $\sigma_\epsilon (\times 10^{-2})$   & 0.908                               & 0.036 & 0.934 & 0.043 \\
+    \bottomrule
+  \end{tabular}
+\end{table}
+
+
+
+
+# Figures {#sec:figs}
+
+Vector graphics do not lose clarity when being scaled. Make your
+figure in pdf format when you first generate it and keep in mind its
+sizes in the article to avoid over-scaling. Do not simply convert a
+jpeg or png image to a pdf.
+
+# Code {#sec:code}
+
+The document class \texttt{jds} provides several commands to decorate
+\begin{itemize}
+  \item inline code, such as \code{print("Hello world!")};
+  \item programming language, such as \proglang{R}, \proglang{Python}, and
+        \proglang{C++};
+  \item software package, such as \pkg{stats}, \pkg{utils}.
+\end{itemize}
+
+
+# Guide for Authors {#sec:guide-authors}
+
+The following requirements must be followed as closely as possible. A
+technically acceptable manuscript that fails to follow these requirements may be
+returned for retyping, leading to delay in publication.We only accept
+submissions in PDF format. The Latex file must be provided after the manuscript
+is accepted.
```

Comment:
Oh they accept only the PDF, maybe that is why the block

```
page: 1  # set the first page number
month: July
year: 2020
volume: xx
issue: xx
doi: "xx.xxxx/xxxxxxxxx"
received: "April, 2020"  # optional: comment it out if no received date
accepted: "May, 2020"  # optional: comment it out if no accepted date
```

is still important to have. 🤔